core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterStructsTest.java

```diff
@@ -41,11 +41,9 @@ private RelToSqlConverterTest.Sql sql(String sql) {
 
   @Test void testNestedSchemaSelectStar() {
     String query = "SELECT * FROM \"myTable\"";
-    String expected = "SELECT \"a\", "
-        + "ROW(ROW(\"n1\".\"n11\".\"b\"), ROW(\"n1\".\"n12\".\"c\")) AS \"n1\", "
-        + "ROW(\"n2\".\"d\") AS \"n2\", "
-        + "\"e\"\n"
-        + "FROM \"myDb\".\"myTable\"";
+    String expected = "SELECT \"a\", ROW(ROW(\"n1\".\"n11\".\"b\"), "
```

Comment:
I would refrain for reformatting the string to make the comparison easier

Reply:
In fact, is there any difference here?

Reply:
Yes, there is no difference, it has been rolled back.